src/main/java/edition/academy/seventh/service/mapper/ItBookMapper.java

```diff
@@ -29,14 +28,14 @@ public ItBookMapper() {
    * @return {@code List<Book>}
    * @throws IOException when JSON isn't read properly
    */
-  public List<Book> mapListOfJson(List<String> listOfJSONBook) throws IOException {
-    List<Book> listOfBooks = new LinkedList<>();
+  public List<DtoBook> mapListOfJson(List<String> listOfJSONBook) throws IOException {
```

Comment:
wg clean code nie powinno pisać się listOf... jako nazwa argumentu. Przecież każdy widzi że to typ interfejsu List. Powinno być coś w stylu booksAsJson